dev_guide/new_app.adoc

```diff
@@ -21,60 +21,62 @@ and converts it into an image that can run inside of a pod. The images are
 deployed via a deployment configuration, and a service is hooked up to the first
 public port of the application.
```

Comment:
I know it's not directly related to this PR/card, but a couple thoughts I had on the "Overview" reading through it again:

1) It struck me that these "Overview" paras start off making it sound like CLI method is all there is (probably because when they were initially written, it was the case). Now that this topic encompasses both the CLI and web console paths, these paras could be a bit more generalized to discuss "creating new apps" overall. The general content/info being described still seems mostly accurate to both paths AFAICT, so maybe just needs some tweaks.

2) Since this is likely to be a topic that is hit early by a developer, probably without having dug a ton through stuff like the Arch dir, it may be nice to link some of these terms to relevant topics that discuss the respective resources/components in more detail. For example: "templates", "images", "Docker registry", "pod", "deployment configuration". Possible targets could be entries in the Arch > Infra Component & Core Objects subdirs.

If not this PR, we should consider for a follow-up.